src/external/rawspeed/rawspeed-identify.cpp

```diff
@@ -119,7 +119,11 @@ int main(int argc, const char* argv[])
 
     fprintf(stderr, "Loading file: \"%s\"\n", argv[1]);
 
+    #if defined (_WIN32)
```

Comment:
It is not because it compiles that it is correct. @Inicola: yes I think we should use _WIN32 only.